app/views/confirmation.njk

```diff
@@ -1,26 +1,26 @@
 {% extends '_layout.njk' %}
-
 {% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
 
 {% block content %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      {{ govukPanel(model) }}
-
-      <p class="govuk-body">We have sent you a confirmation email.</p>
+      <form method="GET" action="/{{referer}}">
+        {{ govukPanel({
+          titleText: "Application submitted",
+          html: "Your reference number<br><strong>HDJ2123F</strong>"
```

Comment:
Could we show the real application number rather than a hard coded value?

Reply:
I think agreementNumber is going to be handled as part of the agreement api. At the moment this is not currently plugged in.